imap_processing/cdf/cdf_attribute_manager.py

```diff
@@ -235,25 +257,54 @@ def get_variable_attributes(self, variable_name: str) -> dict:
         variable_name : str
             The name of the variable to retrieve attributes for.
 
+        check_schema : bool
+            Flag to bypass schema validation.
+
         Returns
         -------
         dict
             Information containing specific variable attributes
             associated with "variable_name".
         """
         output = dict()
-        for attr_name in self.variable_attribute_schema.keys():
-            if variable_name in self._variable_attributes:
-                output = self._variable_attributes[variable_name]
-            elif (
-                variable_name is not None
-                and attr_name in self._variable_attributes["variable_name"]
-            ):
-                output = self._variable_attributes["variable_name"][attr_name]
+        for attr_name in self.variable_attribute_schema["attribute_key"]:
```

Comment:
Ah, yes. So the reason I was doing this is because the default_variable_attribute_schema has a few different sections. There is the "attribute_keys" section where all the attribute names and their information is stored, and near the bottom of the file, there are the sections "data", "support_data", "metadata" that hold different collections of attribute names.

Reply:
You're right, so update the line to be: 
`self.variable_attribute_schema["attribute_keys"].get(variable_name, {})`

Reply:
I think the code has been changed enough that this isn't needed anymore, but please let me know if I am just missing it!

Reply:
Yes! Looks good now, nice job!